packages/colorpickers/src/styled/common/StyledRange.ts

```diff
@@ -60,24 +60,38 @@ const trackLowerStyles = (styles: string, modifier = '') => {
   `;
 };
 
-const colorStyles = (props: IStyledRangeProps & ThemeProps<DefaultTheme>) => {
-  const thumbBackgroundColor = getColorV8('neutralHue', 100, props.theme);
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const thumbBackgroundColor = getColor({ theme, variable: 'background.default' });
   const thumbBorderColor = thumbBackgroundColor;
-  const thumbActiveBackgroundColor = getColorV8('neutralHue', 200, props.theme);
-  const thumbActiveBorderColor = getColorV8('primaryHue', 600, props.theme);
-  const thumbFocusBorderColor = getColorV8('primaryHue', 400, props.theme);
-  const thumbHoverBackgroundColor = thumbActiveBackgroundColor;
+  const thumbActiveBackgroundColor = thumbBackgroundColor;
+  const thumbActiveBorderColor = getColor({
+    theme,
+    variable: 'border.primaryEmphasis'
+  });
+  const thumbFocusBorderColor = thumbActiveBorderColor;
+  const thumbHoverBackgroundColor = getColor({ theme, variable: 'background.subtle' });
   const thumbHoverBorderColor = thumbHoverBackgroundColor;
 
-  return `
-    border-color: ${props.isOpaque && getColorV8('background', 600 /* default shade */, props.theme)};
```

Comment:
With the `isOpaque` option, the border was only used for positioning. The `border-color` must then match with the background color of the the container the ColorPicker with `isOpaque` renders into. That can be problematic in some cases.

![Screenshot 2024-07-17 at 7 21 54 AM](https://github.com/user-attachments/assets/8a8e33fd-4ec5-4889-bf8b-9a360517491f)

So I refactored the CSS to remove it.

Reply:
If this ends up having no issues with the final interaction/element then sounds good 👍